packages/app-desktop/main-html.js

```diff
@@ -39,32 +39,6 @@ window.React = React;
 
 
 const main = async () => {
-	if (bridge().env() === 'dev') {
-		const newConsole = function(oldConsole) {
-			const output = {};
-			const fnNames = ['assert', 'clear', 'context', 'count', 'countReset', 'debug', 'dir', 'dirxml', 'error', 'group', 'groupCollapsed', 'groupEnd', 'info', 'log', 'memory', 'profile', 'profileEnd', 'table', 'time', 'timeEnd', 'timeLog', 'timeStamp', 'trace', 'warn'];
-			for (const fnName of fnNames) {
-				if (fnName === 'warn') {
-					output.warn = function(...text) {
-						const s = [...text].join('');
-						// React spams the console with walls of warnings even outside of strict mode, and even after having renamed
-						// unsafe methods to UNSAFE_xxxx, so we need to hack the console to remove them...
-						if (s.indexOf('Warning: componentWillReceiveProps has been renamed, and is not recommended for use') === 0) return;
-						if (s.indexOf('Warning: componentWillUpdate has been renamed, and is not recommended for use.') === 0) return;
-						oldConsole.warn(...text);
```

Comment:
Based on a code search, all usages of `componentWillReceiveProps` and `componentWillUpdate` in Joplin's components seem to have been renamed. It's possible that one of Joplin's dependencies still uses them, but I haven't noticed related log output.